src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int MAX_NUM = 100;
```

Comment:
Please don't add redundant empty lines to your code.
We don't need them after class declaration or method signature.